asyncio_toolkit/circuit_breaker/coroutine.py

```diff
@@ -18,8 +18,20 @@ def increment(self):
         This method demands that the implementation is responsible for
         getting a storage key from the storage engine.
         """
-
         total = yield from self.storage.increment(self.failure_key, 1)
+        if total == 1:
+            logger.info(
```

Comment:
I think this should be debug ... this will happen a lot and will increase the log amount.

Reply:
Yes, you are right. I fixed this logger level and added a similar log in `CircuitBreaker.increment`.